server/app/views/applicant/ProgramCardsSectionParamsFactory.java

```diff
@@ -60,26 +60,29 @@ public ProgramSectionParams getSection(
       Locale preferredLocale,
       Optional<CiviFormProfile> profile,
       Optional<Long> applicantId,
-      ApplicantPersonalInfo personalInfo) {
-
-    ProgramSectionParams.Builder sectionBuilder =
-        ProgramSectionParams.builder()
-            .setCards(
-                getCards(
-                    request,
-                    messages,
-                    buttonText,
-                    programData,
-                    preferredLocale,
-                    profile,
-                    applicantId,
-                    personalInfo));
+      ApplicantPersonalInfo personalInfo,
+      boolean isMyApplicationsSection) {
```

Comment:
Question (Blocking): Piping a boolean specifically for the My Applications Section feels a bit awkward. What do you think of using an enum to represent different types of sections or card styles instead?

Reply:
Great idea to use an enum.  Added that.